src/components/AutoCompleteTextarea/Textarea.js

```diff
@@ -741,18 +689,6 @@ class ReactTextareaAutocomplete extends React.Component {
                 dropdownScroll={this._dropdownScroll}
               />
             )}
-            {dataLoading && (
```

Comment:
removing this loader, it looks ugly and keeps popping up while typing (even when results don't change)!!